appGeo/src/app/home/home.component.ts

```diff
@@ -70,8 +70,8 @@ export class HomeComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.isKiller = true;   //This information should be received from Database with Player Info!!!
-    this.votingOpen = false; //This information should be received from Database with Game Info!!!
+    //this.isKiller = true;   //This information should be received from Database with Player Info!!!
+    this.votingOpen = true; //This information should be received from Database with Game Info!!!
```

Comment:
Why is this set to true by default?

Reply:
Don't now where else to get the information from voting from @noahklowden, from where should it be extracted?

Reply:
this.votingOpen will automatically be set to "true" when the voting opens and "false" when it closes. It just shouldn't default to true.

Reply:
Made it default to false then in latest commit.